src/app/lecture/lecture.module.ts

```diff
@@ -47,7 +50,8 @@ import { MarkdownEditorComponent } from './knowledge-node/notes/markdown-editor/
     RouterModule,
     FormsModule,
     MaterialModule,
-    DragDropModule
+    DragDropModule,
+    MatButtonToggleModule
```

Comment:
Should be imported and exported from the MaterialModule (imported two lines above).